c7n/resources/workspaces.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import functools
+
+from c7n.actions import ActionRegistry
+from c7n.filters import FilterRegistry
+from c7n.manager import resources
+from c7n.query import QueryResourceManager
+from c7n.tags import universal_augment, register_universal_tags
+from c7n.utils import generate_arn
+
+filters = FilterRegistry('workspaces.filters')
+actions = ActionRegistry('workspaces.actions')
+
+
+@resources.register('workspaces')
+class Workspace(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'workspaces'
+        enum_spec = ('describe_workspaces', 'Workspaces', None)
+        type = 'workspace'
+        name = id = dimension = 'WorkspaceId'
+        filter_name = None
+
+    filter_registry = filters
+    action_registry = actions
+    augment = universal_augment
+    _generate_arn = None
+
+    def __init__(self, data, options):
```

Comment:
since there isn't anything being done in the __init__ override, the whole method can be omitted.